terminus-ui/scss/helpers/_layout.scss

```diff
@@ -120,5 +119,6 @@
   padding: 0;
   pointer-events: none;
   position: absolute;
+  visibility: hidden;
```

Comment:
wonder how this mixin ever worked without setting visibility...

Reply:
It _almost_ worked in my case without it since there is such a tiny tiny area that is visible. You could _just barely_ see a pixel of stuff when I tried, so I just went ahead and added it.

The reason it didn't originally have that is because setting visiblity to hidden removes the element from the accessibility tree which goes against the primary use of the mixin (to hide things visually that need to be there for accessibility).

But getting this out is more important since we don't actually follow strict accessibility here.